### ISSUE TITLE
Correct container settings for DSM 7.

### ISSUE THREAD

**Issue by Alexeyrz** [aldostools/webMAN-MOD#556]
Dear @shawly 
Hello! When running and configuring ps3netsrv on Synology in a Docker container. wMAN Mod can't see games on my NAS. I specified port 38008 in the container settings. I also added the path to the PS3NETSRV folder and linked it to / games. Build ps3netsrv using: shawly /
docker-ps3netsrv. Please tell me how to properly configure the container? I did not find any manuals on the network. 

Is it correct that I specify the path to the PS3NETSRV = /games folder in the container settings (in lower case)? Or maybe I need to specify, for example, the path to a subfolder, like this? PS3NETSRTV/GAMES = /games Or else? 

I have checked several times already.
    1. The checkbox for scan to LAN in the web man settings is checked.
    2. The IP address of my NAS is correct. Port 38008.
    3. Nothing is displayed at the address: http://192.168.88.235/net0. Not folders or files.
    4. In the settings of my NAS:

Path to my PS3NETSRV = /games
At the same time, if I raise ps3netsrv on my PC. Exactly with the same settings in the web man, all games are displayed and at http://192.168.88.235/net0 I see the structure of all folders with games.

**Comment by shawly** [comment]
Sounds like a permission problem, my README should cover every common issue: https://github.com/shawly/docker-ps3netsrv#troubleshooting

**Comment by Alexeyrz** [comment]
> 
> 
> Sounds like a permission problem, my README should cover every common issue: https://github.com/shawly/docker-ps3netsrv#troubleshooting

Hi. You were right. There were problems with the rights to folders. I did it according to your instructions, point D. I was given the following. 


docker exec ps3netsrv ls -l /games
total 0
drwx------ 1 1026 users    0 Feb 25  2021 FIRMWARE
drwx------ 1 1026 users 4352 Jun 13 13:16 GAMES
drwx------ 1 1026 users  342 Feb 25  2021 PKG
drwx------ 1 1026 users    0 Feb 25  2021 PS3ISO


For some reason there is no user ps3netsrv there, but only the Users group on my NAS. And I realized that the group had no rights. Gave the Users group the rights to the game dump folder. And they became visible from my PS3.
